src/dnx.windows/dnx.exe.manifest

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
+    <!-- Compatibility manifests so we get the right behavior out of APIs -->
+    <!-- See: https://msdn.microsoft.com/en-us/library/windows/desktop/dn481241(v=vs.85).aspx -->
+    <compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1"> 
+        <application> 
+            <!-- Windows 10 --> 
```

Comment:
Hmm... that's not great. Is that really the only way we have to determine if we're running on OneCore? If so, I'd be open to removing the Windows 10 entry from here for the short-term while we work out the full plan.